CODEOWNERS

```diff
@@ -18,6 +18,7 @@
 /src/Assets/TestProjects/BlazorHostedRID/ @captainsafia, @pranavkm, @mkArtakMSFT
 /src/Assets/TestProjects/BlazorWasmMinimal/ @captainsafia, @pranavkm, @mkArtakMSFT
 /src/Assets/TestProjects/BlazorWasmWithLibrary/ @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.AoT.Tests/ @captainsafia, @pranavkm, @mkArtakMSFT, @TanayParikh, @javiercn
```

Comment:
I think you can take @captainsafia out from these unless she wants to still be listed.

Reply:
Yep! Feel free to remove me.